src/implementations/twig/components/site-footer/README.md

```diff
@@ -0,0 +1,199 @@
+# ECL Site Footer component
```

Comment:
this component is named "site footer", and not "footer" as previously. Is this a request from COMM?
no real issue for me, but I just wanted to make sure that it was on purpose

Reply:
Yes the new component should be named **site-footer**, see ECL-460